coverage_comment/subprocess.py

```diff
@@ -12,10 +13,11 @@ class GitError(SubProcessError):
     pass
 
 
-def run(*args, **kwargs) -> str:
+def run(*args, path: pathlib.Path, **kwargs) -> str:
```

Comment:
I wonder if it shouldn't be the other way around: fixing places where we call it with a string to use a path instead

Reply:
Sure, the annotation was just to make my point more understandable as it would be one of the two possible ways to solve the problem. But I agree that it would be better to generally use `Path` objects as much as possible when dealing with paths

Reply:
As far as I can tell, path is a string only in tests, it's a Path otherwise. I'll change it in tests.